e2e/test/scenarios/binning/correctness/shared/constants.js

```diff
@@ -94,4 +94,16 @@ export const LONGITUDE_OPTIONS = {
     selected: "20°",
     representativeValues: ["180° W", "160° W", "100° W", "80° W", "60° W"],
   },
+  "Bin every 0.05 degrees": {
```

Comment:
This arrow should be aligned to the right (it's a problem in `master`).

![image](https://github.com/user-attachments/assets/eb8feaab-6fe1-4d63-825f-c0d0e95c3e37)

Reply:
yes, the problem comes from master and I'm working on a fix, but it's not easy